pkg/services/serviceaccounts/api/api.go

```diff
@@ -125,6 +125,42 @@ func (api *ServiceAccountsAPI) CreateServiceAccount(c *contextmodel.ReqContext)
 	return response.JSON(http.StatusCreated, serviceAccount)
 }
 
+func (api *ServiceAccountsAPI) ListServiceAccounts(c *contextmodel.ReqContext) response.Response {
```

Comment:
Where do we use this one?

Reply:
oh we dont actually, this is from another PR. my bad 🥇